spec/index.bs

```diff
@@ -3686,21 +3693,162 @@ GPURenderPipeline includes GPUPipelineBase;
 
 <script type=idl>
 dictionary GPURenderPipelineDescriptor : GPUPipelineDescriptorBase {
+    GPUVertexStateDescriptor vertexState = {};
     required GPUProgrammableStageDescriptor vertexStage;
-    GPUProgrammableStageDescriptor fragmentStage;
-
     required GPUPrimitiveTopology primitiveTopology;
```

Comment:
probably index format should be in the primitive assembly stage. see #767

Reply:
that's what #1307 does, which we should focus on before continuing on this one

Reply:
I just recorded this as a quick note during the editing meeting just to make sure we didn't forget about it when you created #1307. So marking as resolved now.